src/ipynb/spark_ml.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.    # Disable the warning for too many open figures
</pre><p>What does this mean? Is it crucial that it is explained?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>